### PR TITLE
Circstore 93 Add new form fields for Request Management subsections

### PR DESCRIPTION
**Pull request by kostyantyn-kh**

### CODE REVIEW

File: ramls/loan-policy.json [hunk 1]
```diff
@@ -85,6 +85,69 @@
...
+      "properties": {
+        "recalls": {
+          "type": "object",
+          "properties": {
+            "alternateGracePeriod": {
+              "type": "object",
+              "$ref": "period.json",
+              "description": "Alternate grace period for recalled items"
+            },
+            "minLoanPeriod": {
```
[the remark this answers is not in the document]
Comment (kostyantyn-kh) [reply ramls/loan-policy.json]: Used short form of 'minimum' - 'min'

File: ramls/loan-policy.json [hunk 2]
```diff
@@ -85,6 +85,69 @@
         }
       }
     },
+    "requestManagement": {
+      "type": "object",
```
Comment (marcjohnson-kint) [inline ramls/loan-policy.json]: Each of the object properties needs to include `"additionalProperties": false` to disallow additional properties within it

File: ramls/loan-policy.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (marcjohnson-kint) [reply ramls/loan-policy.json]: I understand that the shorter form was used. I'm curious as to why it was preferred over the full word (which for me is more readable)?

[the remark this answers is not in the document]
Comment (marcjohnson-kint) [reply ramls/loan-policy.json]: Or alternatively, to match with the UI `minimumGuaranteedLoanPeriod`?

File: ramls/loan-policy.json (hunk 2 above)
Comment (kostyantyn-kh) [reply ramls/loan-policy.json]: added

File: ramls/loan-policy.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (kostyantyn-kh) [reply ramls/loan-policy.json]: renamed

File: ramls/loan-policy.json (hunk 2 above)
Comment (marcjohnson-kint) [reply ramls/loan-policy.json]: It appears they were added to all of the object properties. Sorry, that was likely my poor description. If we add them to the existing objects, that would be a breaking change. I meant only the new object properties added during this change. Apologies :-(

Comment (kostyantyn-kh) [reply ramls/loan-policy.json]: removed for existing objects

Comment (marcjohnson-kint) [reply ramls/loan-policy.json]: Thanks

File: ramls/loan-policy.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (marcjohnson-kint) [reply ramls/loan-policy.json]: Thank you

**Pull request merged**